test/Garnet.test/RespPubSubTests.cs

```diff
@@ -97,5 +98,198 @@ public void BasicPSUBSCRIBE()
             }
             sub.Unsubscribe(channel);
         }
+
+        [Test]
+        public void BasicPUBSUB_CHANNELS()
+        {
+            using var subRedis = ConnectionMultiplexer.Connect(TestUtils.GetConfig());
+            using var redis = ConnectionMultiplexer.Connect(TestUtils.GetConfig());
+            var sub = subRedis.GetSubscriber();
+            var db = redis.GetDatabase(0);
+            var server = redis.GetServers()[0];
+
+            ManualResetEvent evt = new(false);
+            var isMessagesASubed = false;
+            var isMessagesBSubed = false;
+            var channelA = "messagesAtest";
+            var channelB = "messagesB";
+
+            sub.Subscribe(RedisChannel.Literal(channelA), (channel, message) =>
+            {
+                isMessagesASubed = true;
+                if (isMessagesBSubed)
+                    evt.Set();
+            });
+
+            sub.Subscribe(RedisChannel.Literal(channelB), (channel, message) =>
+            {
+                isMessagesBSubed = true;
+                if (isMessagesASubed)
+                    evt.Set();
+            });
+
+            // Doing publish to make sure the channel is subscribed
```

Comment:
Is this bug still valid in the latest version of SE.Redis?

Reply:
I just tested it, and it's still valid. IMO, this is an expected behaviour from SE.Redis and it will never be changed.